elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/ElasticsearchEnvironment.java

```diff
@@ -223,6 +224,21 @@
 	 */
 	public static final String SCROLL_TIMEOUT = "elasticsearch.scroll_timeout";
 
+	/**
+	 * Property for specifying the behaviour when Elasticsearch encounters a previously unknown field in a document.
```

Comment:
I mean: it's not clear if this is to configure the "behaviour of Elasticsearch" or of Hibernate Search